test/parser.test.ts

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+"use strict";
+import {
+  parseRamlFile,
+  getAllDataTypes,
+  getApiName,
+  resolveApiModel
+} from "../src/parser";
+
+import {
+  WebApiBaseUnitWithDeclaresModel,
+  WebApiParser,
+  model,
+  webapi,
+  WebApiBaseUnitWithEncodesModel
+} from "webapi-parser";
+
+import { expect, default as chai } from "chai";
+import path from "path";
+import chaiAsPromised from "chai-as-promised";
+import _ from "lodash";
+
+before(() => {
+  chai.use(chaiAsPromised);
```

Comment:
chai-as-promised is never actually used. Tests involving promises should either be cleaned up and use it, or they should be converted to async functions and chai-as-promised can be removed.

Reply:
done